recipes/nibabel/run_test.py

```diff
@@ -0,0 +1,14 @@
+import nose
+import sys
+import re
+
+splatform = sys.platform
+if splatform.startswith('win32'):
+    # Installation of data files for script testing a bit rickety on Appveyor:
+    config = nose.config.Config(verbosity=2,
+                                exclude=[re.compile("test_scripts"),
+                                         re.compile("test_dft")])
+else:
+    config = nose.config.Config(verbosity=2)
```

Comment:
Would you mind taking the verbosity down? The logs are a bit too long for the CI services to render, and all the conda recipe should really be doing is verifying the build was good, not acting as extra coverage for the package itself so we don't really need to know *which* tests are passing/failing until we want to debug it.